core/lib/contracts/src/lib.rs

```diff
@@ -40,8 +40,12 @@ const LOADNEXT_CONTRACT_FILE: &str =
 const LOADNEXT_SIMPLE_CONTRACT_FILE: &str =
     "etc/contracts-test-data/artifacts-zk/contracts/loadnext/loadnext_contract.sol/Foo.json";
 
-fn read_file_to_json_value(path: impl AsRef<Path>) -> serde_json::Value {
-    let zksync_home = std::env::var("ZKSYNC_HOME").unwrap_or_else(|_| ".".into());
+fn home_path() -> PathBuf {
```

Comment:
Like with contract verifier, relying on the workspace looks suboptimal. It basically makes this crate (and all crates depending on it) non-publishable. I guess we could use as a stopgap solution, but IMO a longer-term solution would be to include the contracts into the library itself in compile time via something like `include_bytes!`. Still leaves the open question of where to get contract bytecode from, of course; in an ideal world, it would be a part of the build script. IMO, it would be acceptable to cheat in this build script for now, assuming that it's invoked in the workspace with the contracts already built; compare this with specifying `ROCKSDB_LIB_DIR` when building `rocksdb` crate. Would still be a significant improvement over the current runtime trickery.